docs/linux.md

```diff
@@ -49,16 +50,18 @@ User=ollama
 Group=ollama
 Restart=always
 RestartSec=3
+Environment="PATH=$PATH"
 
 [Install]
 WantedBy=default.target
 ```
 
-Then start the service:
+Then start and check the service:
 
 ```bash
 sudo systemctl daemon-reload
 sudo systemctl enable ollama
+sudo systemctl status ollama
```

Comment:
I don't think this line is required. The "Start Ollama" section below should cover it.
```suggestion
```

Reply:
removed with https://github.com/ollama/ollama/pull/6430/commits/2abc330d96faffff6d11dc1a3128205753c5d8cc